distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -480,6 +486,214 @@ public void testBasicRestAccess() throws Exception {
+                "TLS_RSA_WITH_AES_128_CBC_SHA",
+                "TLS_ECDH_ECDSA_WITH_AES_128_CBC_SHA",
+                "TLS_ECDH_RSA_WITH_AES_128_CBC_SHA",
+                "TLS_DHE_RSA_WITH_AES_128_CBC_SHA",
+                "TLS_DHE_DSS_WITH_AES_128_CBC_SHA",
+                "TLS_ECDHE_ECDSA_WITH_RC4_128_SHA",
+                "TLS_ECDHE_RSA_WITH_RC4_128_SHA",
+                "SSL_RSA_WITH_RC4_128_SHA",
+                "TLS_ECDH_ECDSA_WITH_RC4_128_SHA",
+                "TLS_ECDH_RSA_WITH_RC4_128_SHA",
+                "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384",
+                "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384",
+                "TLS_RSA_WITH_AES_256_GCM_SHA384",
+                "TLS_ECDH_ECDSA_WITH_AES_256_GCM_SHA384",
+                "TLS_ECDH_RSA_WITH_AES_256_GCM_SHA384",
+                "TLS_DHE_RSA_WITH_AES_256_GCM_SHA384",
+                "TLS_DHE_DSS_WITH_AES_256_GCM_SHA384",
+                "TLS_RSA_WITH_AES_128_GCM_SHA256",
+                "TLS_ECDH_ECDSA_WITH_AES_128_GCM_SHA256",
+                "TLS_ECDH_RSA_WITH_AES_128_GCM_SHA256",
+                "TLS_DHE_RSA_WITH_AES_128_GCM_SHA256",
+                "TLS_DHE_DSS_WITH_AES_128_GCM_SHA256",
+                "TLS_ECDHE_ECDSA_WITH_3DES_EDE_CBC_SHA",
+                "TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA",
+                "SSL_RSA_WITH_3DES_EDE_CBC_SHA",
+                "TLS_ECDH_ECDSA_WITH_3DES_EDE_CBC_SHA",
+                "TLS_ECDH_RSA_WITH_3DES_EDE_CBC_SHA",
+                "SSL_DHE_RSA_WITH_3DES_EDE_CBC_SHA",
+                "SSL_DHE_DSS_WITH_3DES_EDE_CBC_SHA",
+                "SSL_RSA_WITH_RC4_128_MD5",
+                "TLS_EMPTY_RENEGOTIATION_INFO_SCSV");
+
+        String[] supportedHttpProtocols = {"https"};
+
+        String originalCiphers = System.getProperty("https.cipherSuites");
+        System.setProperty("https.cipherSuites", StringUtils.join(disallowedCipherSuites, ","));
+
+        SSLContext sslContext = SSLContext.getDefault();
+        SSLSocketFactory sslSocketFactory = new SSLSocketFactory(sslContext,
+                supportedHttpProtocols,
+                disallowedCipherSuites.toArray(new String[disallowedCipherSuites.size()]),
+                new AllowAllHostnameVerifier());
+
+        RestAssuredConfig config = new RestAssuredConfig();
+        config.sslConfig(SSLConfig.sslConfig()
+                .sslSocketFactory(sslSocketFactory));
+
+        String url = SERVICE_ROOT.getUrl() + "/catalog/query?q=*&src=local";
+
+        try {
+            configureRestForBasic(SDK_SOAP_CONTEXT);
+            waitForSecurityHandlers(url);
+            given().config(config)
+                    .auth()
+                    .basic("admin", "admin")
+                    .when()
+                    .get(url)
+                    .then()
+                    .assertThat()
+                    .statusCode(401);
+        } finally {
+            System.setProperty("https.cipherSuites", originalCiphers);
```

Comment:
❗️ Same regarding runtime exceptions that may be thrown above your `try` block. Move lines 665-677 inside the block to be safe.